### DISCUSSION TITLE
Is there any way to render an extremely high text?

### DISCUSSION THREAD

**Discussion by Redgeioz** [pixijs/pixijs#8774] [Q&A] [answered]
Hi, I found something interesting.

There is an extremely high text "ฮึืึืึืึืึืึืึืึื". I tried it in pixijs but failed to render it correctly.
![image](https://user-images.githubusercontent.com/52198272/197394572-978cf904-ba0b-41b4-9b5f-7d9367abe085.png)
So is there any way?
I don't think I really need it. Just curious.

**Comment by SuperSodaSea** [comment, marked as answer]
Current [`TextMetrics.measureText`](https://github.com/pixijs/pixijs/blob/3ba43e50030641e2cb028c7c265e80ce0fdf6fff/packages/text/src/TextMetrics.ts#L156-L157) implementation simply calculate `height` as something like `lineHeight * lineCount`, so the extra parts at the top (maybe bottom) are trimed. I think this problem can be fixed, but need to change a bunch of things in `Text` and `TextMetrics`.

**Reply by Redgeioz** [reply]
Thanks for your answer! If so pixijs would be better.
